lib/wallaroo/core/topology/_test_step_seq_id_generator.pony

```diff
@@ -74,7 +74,6 @@ class iso _TestCurrentDoesNotIncrement is UnitTest
     let gen = StepSeqIdGenerator
 
     h.assert_eq[SeqId](0,gen.current_seq_id())
-    h.assert_eq[SeqId](0,gen.current_seq_id())
```

Comment:
what's this change? not sure how it is related to everything else?

Reply:
Yep, accidental removal, adding it back.